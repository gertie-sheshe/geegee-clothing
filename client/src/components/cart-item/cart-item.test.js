import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

describe('CartItem component tests', () => {
  const mockItem = {
    imageUrl: 'www.testImage.com',
    price: 10,
    name: 'hats',
    quantity: 2,
  };

  it('Should render CartItem component', () => {
    expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
  });
});
