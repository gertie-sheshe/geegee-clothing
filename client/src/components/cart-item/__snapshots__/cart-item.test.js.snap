// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component tests Should render CartItem component 1`] = `ShallowWrapper {}`;
